### PR TITLE
Add git-try-push Action

### PR DESCRIPTION
**Pull request by dawidd6**
An Action to perform `git push` with specified number of tries. If `git push` fails, sleep a bit and then run `git pull --rebase`.

https://github.com/Homebrew/brew/issues/7805

### CODE REVIEW

**Comment by dawidd6** [comment]
This is yet to be finished. Would mark the PR as draft, but I can't see the button for it (disappeared after redesign?).

File: git-try-push/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Git try push Github Action
```
Comment (MikeMcQuaid) [inline git-try-push/README.md:1]:
```suggestion
# Git try push GitHub Action
```

**Comment by jonchang** [comment]
Looks good. Feel free to merge this and update the other repositories to use this when you feel like it's ready!

**Comment by dawidd6** [comment]
Merging as is. Will fix stuff later if needed.

**Pull request merged**